environments/servlet/tests/jetty/pom.xml

```diff
@@ -314,9 +314,9 @@
                         <groupId>org.apache.maven.plugins</groupId>
                         <artifactId>maven-surefire-plugin</artifactId>
                         <configuration>
-                            <excludes>
-                                <exclude>%regex[.*servlet/test/(?!se).*]</exclude>
-                            </excludes>
+                            <includes>
```

Comment:
Weld/WeldContainer are not broken - it's the same problem, but instead of `BeanManager`, either `Weld` (builder) or `WeldInstance` (container instance) is passed from Weld SE. Just add `context.setInitParameter(org.jboss.weld.Container.CONTEXT_ID_KEY, contextId)` to the _if_ block mentioned above.